base/src/main/proto/spine/options.proto

```diff
@@ -754,6 +766,23 @@ message GoesOption {
     string msg_format = 2;
 }
 
+// Defines options of a repeated field that defines a logic
+message OnDuplicate {
```

Comment:
`OnDuplicate` may itself be an enum. Then we won't be forced to write the field name when using the option.